modules/roles_profiles/manifests/profiles/microsoft_tools.pp

```diff
@@ -3,56 +3,78 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 class roles_profiles::profiles::microsoft_tools {
+  case $facts['os']['name'] {
+    'Windows': {
+      include win_os_settings::powershell_profile
+      class { 'win_packages::performance_tool_kit':
+        moz_profile_source => lookup('win-worker.mozilla_profile.source'),
+        moz_profile_file   => lookup('win-worker.mozilla_profile.local'),
+      }
 
-    case $facts['os']['name'] {
-        'Windows': {
-
-            # These may chnage for diffrent versions of Windows
-            # TODO: Research why we are using a pinned version and add comment why.
-            # Original source 'https://hg.mozilla.org/mozilla-central/raw-file/360ab7771e27/toolkit/components/startup/mozprofilerprobe.mof'
-            # When pulling from a HG repo Puppet see it the file as a new file one ach run
-            # In this case triggers the exec and adds to the local WMI repo each time
-            # For now pulling from S3
-
-            include win_os_settings::powershell_profile
+      case $facts['custom_win_os_version'] {
```

Comment:
We can combine some of the lines here like `'win_10_2004', 'win_11_2009' : {` . 

[For reference.](https://puppet.com/docs/puppet/5.5/lang_conditional.html)